### PR TITLE
MOR-412 WCAG compliance

### PR DESCRIPTION
**Pull request by sulaiman-rtf**

### CODE REVIEW

File: src/components/dashboard/PreviewItem.vue [hunk 1]
```diff
@@ -21,8 +21,8 @@
       </div>
       <div v-else class="regular" :style="'background: '+colors.default_button">
         <div class="imageContainer" :style="'border-color: '+colors.default_button">
-          <b-img :src="getIconUrl(item.configuration.image_url)"/>
-        </div>`
+          <b-img :src="getIconUrl(item.configuration.image_url)" alt=""></b-img>
```
Comment (stegru) [inline src/components/dashboard/PreviewItem.vue:24]: A better way could be to change the `alt` attribute to `:alt="item.configuration.label"` or `:alt="item.configuration.label + ' logo'"`

File: src/components/dashboardV2/BarItemLink.vue [hunk 2]
```diff
@@ -6,7 +6,7 @@
                @mouseleave="$emit('mouseleave', {data: barItem})"
                :to="editLink"
   />
-  <b-link v-else
+  <b-botton v-else
```
Comment (stegru) [inline src/components/dashboardV2/BarItemLink.vue:9]: should be `b-button`

File: src/views/focused/FocusedButtonCatalog.vue [hunk 3]
```diff
@@ -15,10 +15,10 @@
           <h3 :class="'header_' + subkind">{{buttonGroup.title}}</h3>
           <ul class="ButtonsCatalogEntries">
             <li v-for="(button, buttonKey) in buttonGroup.items" :key="buttonKey" class="buttonsCatalogEntry">
-              <b-link v-if="currentlyActiveButton != buttonKey" @click="buttonActivated(buttonKey, button)" :style="'color: ' + (button.configuration.color || colors.blue) + ';'"  class="buttonsCatalogEntry nonExpandedCatalogEntry">
-                <b-img v-if="button.configuration.image_url" :src="getIconUrl(button.configuration.image_url)" />
+              <b-botton v-if="currentlyActiveButton != buttonKey" @click="buttonActivated(buttonKey, button)" :style="'color: ' + (button.configuration.color || colors.blue) + ';'"  class="buttonsCatalogEntry nonExpandedCatalogEntry">
```
Comment (stegru) [inline src/views/focused/FocusedButtonCatalog.vue:18]: `button`

**Review by ssanusi — COMMENTED** [review]
all the changes have been effected

**Review by stegru — APPROVED** [review]

**Pull request merged**